dragon/console.rb

```diff
@@ -553,6 +553,12 @@ def process_inputs args
         prompt.delete
       elsif args.inputs.keyboard.key_down.tab
         prompt.autocomplete
+      elsif args.inputs.keyboard.control and (args.inputs.keyboard.key_down.plus or args.inputs.keyboard.key_down.equal_sign)
+        adjust_font_size +1
+        prompt.backspace
```

Comment:
What is this backspace for?

Reply:
Without it, the character "-", "+" or "=" is added to the prompt.

Alternatively, we could look for the combos before [this line](https://github.com/DragonRuby/dragonruby-game-toolkit-contrib/blob/bbde456704114dc87d1f45ea39a9f21505ffcfb0/dragon/console.rb#L484), and add `args.inputs.text.clear` to each branch.

Reply:
It happens to update the cursor's screen position, too, but it's better for that to be done by Prompt#font_style=, hence the new commit